modules/SpyUtils.js

```diff
@@ -27,7 +27,11 @@ export const createSpy = (fn, restore = noop) => {
 
   let targetFn, thrownValue, returnValue
 
-  const spy = function spy() {
+  const spy = new Function('spy', `return function(${ // eslint-disable-line no-new-func
```

Comment:
this is actually only necessary in engines that do not have a configurable function length - ie, this should probably be encapsulated in a function that detects this, and uses `Object.defineProperty` to set the length when possible.